translations/strings.yaml

```diff
@@ -63,3 +63,21 @@ settingsDev:
   useStagingTitle: Use Staging Servers
   useStagingSubtitle: Staging services are used to test features before they go live to production
   restartRequired: VPN must be restarted to take effect
+
+splittunnel:
+  missingAppActionButton:
+    value:
+      - Add apps manually
+    comment:
+      - Displayed in a notification as "call to action" when apps are missing from the split-tunnel settings.
+  missingAppMultiple: 
+    value:
+      - \%0 apps missing from list.
+    comment:
+      - Apps listed in the split-tunnel settings. %0 is the amount of apps missing
```

Comment:
Nit: `%0 is the number of missing apps`. I would also add a bit of help for locales with complex plurals

```
- Apps listed in the split-tunnel settings. %0 is the number of missing apps.
- If necessary, this can be translated as "Apps missing from list: %0".
```